tsconfig.base.json

```diff
@@ -29,7 +29,7 @@
       "@keymanapp/keyman": ["./web" ],
       "@keymanapp/models-templates": ["./web/src/engine/predictive-text/templates/"],
       "@keymanapp/models-wordbreakers": ["./web/src/engine/predictive-text/wordbreakers"],
-      "@keymanapp/web-utils": ["./common/web/utils"],
+      "@keymanapp/web-utils": ["./web/src/engine/common/web-utils"],
```

Comment:
Seems to me that @keymanapp/web-utils will become unnecessary as a published package as it will be part of the web source itself